core/basic-authorship/src/basic_authorship.rs

```diff
@@ -214,7 +214,7 @@ impl<Block, C, A> Proposer<Block, C, A>	where
 				let pending_iterator = self.transaction_pool.ready();
 
 				for pending in pending_iterator {
-					// TODO [ToDr] Probably get rid of it, and validate in runtime.
+					// FIXME [ToDr] Probably get rid of it, and validate in runtime.
```

Comment:
```suggestion
```

Reply:
It's part of #1354, will prep a PR soon.